lib/hook/class.tx_content_replacer.php

```diff
@@ -191,7 +207,7 @@ public function getHeader(tslib_cObj $contentObject) {
 	 * @return string
 	 */
 	public function getPurchaseUrl(tslib_cObj $contentObject) {
-		ini_set('display_errors', 'on');
+
 		// @TODO : check if block with such ID exists
```

Comment:
No TODOs in pull requests. Either fix it or create a ticket for it to be fixed later.